envpool/classic_control/pendulum.h

```diff
@@ -77,19 +77,18 @@ class PendulumEnv : public Env<PendulumEnvSpec> {
   void Step(const Action& action) override {
     done_ = (++elapsed_step_ >= max_episode_steps_);
     float act = action["action"_];
-    double u = act < -kMaxTorque  ? -kMaxTorque
-               : act > kMaxTorque ? kMaxTorque
-                                  : act;
+    double u =
+        act < -kMaxTorque ? -kMaxTorque : act > kMaxTorque ? kMaxTorque : act;
```

Comment:
that is what 'make format' did

Reply:
maybe there is some conflict between 'make format' and 'make lint'

Reply:
I think it's because of version mismatch?